cidc_api/models/files/facets.py

```diff
@@ -236,6 +236,7 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
             "Data files from TCRseq analysis indicating TRA & TRB clones UMI counts",
         ),
     },
+    "ELISA": {"All Files": FacetConfig(["/elisa/assay.xlsx"])},
```

Comment:
Would it be appropriate to label this facet `"Source"`? Or, if we only expect one facet for `"ELISA"` going forward, we could use a non-nested facet like this:
```python
"ELISA": FacetConfig(["/elisa/assay.xlsx"])
```
(see the `clinical_facets` as examples below)

Reply:
would [olink](https://github.com/CIMAC-CIDC/cidc-api-gae/blob/master/cidc_api/models/files/facets.py#L206) also want this type of labeling?

Reply:
Maybe - we might want multiple nested facets to distinguish between the "raw" npx and ct files and the combined biomarker file you're introducing.

Reply:
in next commit